dashboard/test/models/user_level_test.rb

```diff
@@ -51,6 +54,36 @@ class UserLevelTest < ActiveSupport::TestCase
     assert_equal UserLevel.by_lesson(lesson), [first_user_level, second_user_level]
   end
 
+  test "driver?" do
+    assert_equal false, @unpaired_user_level.driver?
+    assert_equal true, @driver_user_level.driver?
+    assert_equal false, @navigator_user_level.driver?
+  end
+
+  test "navigator?" do
+    assert_equal false, @unpaired_user_level.navigator?
+    assert_equal false, @driver_user_level.navigator?
+    assert_equal true, @navigator_user_level.navigator?
+  end
+
+  test "paired?" do
+    assert_equal false, @unpaired_user_level.paired?
+    assert_equal true, @driver_user_level.paired?
+    assert_equal true, @navigator_user_level.paired?
+  end
+
+  test "driver" do
+    assert_equal nil, @unpaired_user_level.driver
+    assert_equal @driver, @driver_user_level.driver
+    assert_equal @driver, @navigator_user_level.driver
+  end
+
+  test "navigators_names" do
+    assert_equal nil, @unpaired_user_level.navigators_names
```

Comment:
Would it be better if this method just returned an empty array instead of nil? So that the type of the thing it returns is predictable?

Reply:
Good question.  I chose nil here because I was trying to keep it consistent with `driver` which returns nil when the user_level is unpaired.  Thoughts?

Reply:
I slightly learn towards a preference to return an empty array for consistency, but this is really a nit pick. Feel free to leave as-is

Reply:
I think I favor the consistency with `driver` more so I'll leave this as-is.